samples/bluetooth/fast_pair/locator_tag/src/fp_adv.c

```diff
@@ -493,32 +528,38 @@ uint8_t app_fp_adv_id_get(void)
 	return fp_adv_param.id;
 }
 
-int app_fp_adv_uninit(void)
+int app_fp_adv_init(void)
 {
 	int err;
+	int trigger_cnt;
```

Comment:
Shouldn't it be `size_t` ?

Reply:
It doesn't really matter which type we will use here.
Doubt that the section size would be > `2^31-1`.
In the zephyr itself once is used `int`, once `size_t` and once `uint32_t`. 
Let's leave it as is.

Reply:
Ok, I would not insist